test/test_bpmn_cwp.py

```diff
@@ -422,3 +682,38 @@ def test_promela_and_ltl():
 
     with open("promela_ltl.pml", "w") as f:
         f.write(output_str)
+
+
+def test_cwp_write_init_states():
+    with patch.object(CwpLtlVisitor, "write_line") as mock_write:
+        cwp = generate_mock_cwp()
+
+        symbol_table = MagicMock()
```

Comment:
This mock use is good practice. We're not testing the symbol table, and we don't need functionality yet, and when we do, a mock still makes more sense.